tests/native_api_tests/test_api_model.cpp

```diff
@@ -1063,7 +1063,7 @@ TEST_CASE("API Model") {
         }
     }
 
-    SUBCASE("Current sensor is experimental") {
+    SUBCASE("Current sensor for NRSE is experimental: not implemented") {
```

Comment:
Nitpick: I would rephrase positively.
```suggestion
    SUBCASE("Current sensor for ILSE is implemented") {
```

Reply:
This will probably give the wrong impression

Reply:
Why? What do you mean?

Reply:
Of all the existing features, why specifically have a test for ILSE current sensor? This particular test is meant to make sure the yet-to-implement-feature in speak do not escape. Since this is not a promotion sentence, the positive rephrasing is not needed and not accurate.